src/middlewared/middlewared/plugins/jail.py

```diff
@@ -473,3 +475,74 @@ def _import(self, job, jail):
         IOCImage().import_jail(jail)
 
         return True
+
+    @accepts()
+    def get_plugin_versions(self):
+        """
+        Fetches a list of pkg's from the http://pkg.cdn.trueos.org/iocage/
+        repo and returns a dictionary for each plugin
+        """
+        r_pkgs = urllib.request.urlopen('http://pkg.cdn.trueos.org/iocage/All')
+        r_plugins = json.load(
+            urllib.request.urlopen(
+                'https://raw.githubusercontent.com/freenas/'
+                'iocage-ix-plugins/master/INDEX')
+        )
+        pkgs = {
+            'bruserver': ('N/A', '1'),
+            'sickrage': ('Git Branch - master', '1')
+        }
+        pkg_dict = {}
+        plugin_pkgs = {}
+        plugin_manifests = []
+        plugin_translation = {
+            'rslsync': 'btsync',
+            'emby-server': 'emby',
+            'transmission-daemon': 'transmission',
+            'backuppc4': 'backuppc',
+            'nextcloud': 'nextcloud-php56',
+            'tt-rss': 'ttrss',
+            'deluge-cli': 'deluge',
+            'madsonic-standalone': 'madsonic',
+            'subsonic-standalone': 'subsonic',
+            'quassel-core': 'quasselcore',
+            'linux-crashplan': 'crashplan'
+        }
```

Comment:
That translation seems very weird to me. Can we make it part of the manifest somehow?